presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -308,6 +309,11 @@ private boolean isTablesEnumeratingTable(SchemaTableName schemaTableName)
         return ImmutableSet.of(TABLE_COLUMNS, TABLE_VIEWS, TABLE_TABLES, TABLE_TABLE_PRIVILEGES).contains(schemaTableName);
     }
 
+    private boolean isColumnsEnumeratingTable(SchemaTableName schemaTableName)
```

Comment:
from cmt msg

> this is only beneficial
for columns enumerating metadata table since it can narrow
down results early.

i guess there is a type: "for columns" -> ...?